pkg/process/net/common_windows.go

```diff
@@ -28,8 +28,14 @@ const (
 	procStatsURL = "http://localhost:3333/" + string(sysconfig.ProcessModule) + "stats"
 	// pingURL is not used in windows, the value is added to avoid compilation error in windows
 	pingURL = "http://localhost:3333/" + string(sysconfig.PingModule) + "/ping/"
+
+	// SystemProbeProductionPipeName is the production named pipe for system-probe
+	SystemProbeProductionPipeName = `\\.\pipe\dd_system_probe`
 )
 
+// SystemProbePipeName is the effective named pipe path for system-probe
+var SystemProbePipeName = SystemProbeProductionPipeName
```

Comment:
It seems that continuing to use a configuration value for the pipe name might be a good idea. That would likely prevent the need for exported globals. WDYT?

Reply:
The business case as of now is that the named pipe will not be externally configurable.  
It is also not reasonable/user-friendly to reuse the the existing config field "system_probe_config.sysprobe_socket" for a named pipe path.
AFAIK, the main reason for the configurable field "system_probe_config.sysprobe_socket" is to address port conflicts, which with the dedicated named pipe will be resolved.
The only use case that requires a named pipe override is for a local e2e test running SxS with an existing Datadog install.

Reply:
Having an exported global that is overridden is not a good abstraction, and a code smell.  I agree that "socket" could be a confusing name for the configuration variable. We have a lot of configuration variables that are not documented and realistically not intended to be changed by customers, but they allow us to retain that flexibility (such as for tests).

Reply:
I removed the exported global and instead introduce a function OverrideSystemProbeNamedPipePath, which is used by tests only.